lib/env.c

```diff
@@ -100,6 +101,37 @@ static struct ul_env_list *env_list_add(struct ul_env_list *ls0,
 	return ls;
 }
 
+/*
+ * Calls getenv() for each name (comman separated) in @str and add result to the list
+ */
+struct ul_env_list *env_list_add(struct ul_env_list *ls, const char *str)
+{
+	if (!str)
+		return ls;
+
+	if (strchr(str, ',')) {
+		char **all, **name;
+
+		all = strv_split(str, ",");
```

Comment:
Does this split really belong in here? Can't it be punted up to the caller?

Reply:
It will be used in more places, so I'd like to avoid keeping it in callers, but it can be implemented in a separate function (something like env_list_import_names()).